docs/user-guide/tuning.md

```diff
@@ -113,19 +113,25 @@ The advice for tuning the performance of `MPI_Send` is
 #### Setting the eager limit on ARCHER2
 
 On ARCHER2, things are a little more complicated. Although the eager
-limit defaults to 16K bytes, messages up to 256K are sent
+limit defaults to 16KiB, messages up to 256KiB are sent
 asynchronously because they are actually sent as a number of smaller
 messages.
 
 To send even larger messages asynchronously, alter the value of
-`FI_OFI_RXM_SAR_LIMIT` in your SLURM script, e.g. to set to 512K:
+`FI_OFI_RXM_SAR_LIMIT` in your job submission script script, e.g. to set to 512KiB:
 
     export FI_OFI_RXM_SAR_LIMIT=524288
 
+You can also change the size of the smaller messages that larger ones
+are broken up into by altering the value of `FI_OFI_RXM_BUFFER_SIZE`
```

Comment:
Might be the following sentence more readable? _"You can also change the size of the smaller messages, **causing** that larger ones are broken up into, by altering the value of [...]"_

Reply:
I've suggested a fix